samples/c2pa/monolithic/c2pa-demo-monolithic.html

```diff
@@ -32,7 +37,7 @@
         
         function init() {
             var video,
-                url = "../../microsoft/video_samples/monolithic/provenanceoutput_progressivedownload.mp4";
+              url = "../../microsoft/video_samples/monolithic/provenanceoutput_progressivedownload.mp4";
```

Comment:
Just a reminder to not forget about updating this URL to the public one.

Reply:
yep - the public link doesn't have the new menu assertions or error segments- so I've been keeping this for testing